src/vs/workbench/contrib/terminal/browser/media/CodeTabExpansion.psm1

```diff
@@ -0,0 +1,57 @@
+# TODO: Dynamically enable depending on quality?
+Microsoft.PowerShell.Core\Register-ArgumentCompleter -CommandName "code","code-insiders" -Native -ScriptBlock {
+	param($wordToComplete, $commandAst, $cursorPosition)
+
+	# TODO: These tooltips could be localized?
+	# TODO: Context aware suggestions
+	# TODO: Auto-generate this
+	# TODO: Subcommands
+	@(
```

Comment:
Does this file get signed so that users whose ExecutionPolicy is set to the default value (usually AllSigned) will work?